src/sfctl/custom_chaos.py

```diff
@@ -43,6 +72,60 @@ def start( #pylint: disable=too-many-arguments,too-many-locals
     entry specifies as a key the application type name and as  a value an
     integer that represents the MaxPercentUnhealthyApplications percentage
     used to evaluate the applications of the specified application type.
+    :param str context: JSON encoded map of (string, string) type key-value
```

Comment:
Can you instead move this help string into a new file in `src/sfctl/helps`. That's where all new docstrings should go since it supports more features and YAML formatting.

Reply:
Moved

Reply:
I think it should still work when you remove it from here... can you confirm? Thanks! :)